Makefile

```diff
@@ -39,7 +43,13 @@ BOOTARGS := console=hvc0 earlycon=sbi
 NCPU ?= 1
 MEM_SIZE ?= 4096
 EXTRA_QEMU_ARGS ?=
+
+ifneq ($(VECTORS),)
+CPU_ARGS := rv64,v=on,vlen=256,elen=64,x-aia=true,sscofpmf=true
+else
 CPU_ARGS := rv64,x-aia=true,sscofpmf=true
+endif
```

Comment:
instead of defining the common args twice, you could do something like:

```
CPU_ARGS := rv64,x-aia=true,...
ifneq ($(VECTORS),)
CPU_ARGS += ,v=on,...
endif
```

Reply:
I pulled out the CPU_TYPE to make easier to change it to something more specific if needed